charts/devnet/templates/explorer.yaml

```diff
@@ -6,11 +6,15 @@ metadata:
   name: explorer
   labels:
     app.kubernetes.io/name: explorer
+  annotations:
+    cloud.google.com/backend-config: '{"ports": {
+    "http": "explorer-hc-config"
+    }}'
 spec:
   {{- if .Values.explorer.external }}
   type: LoadBalancer
   {{- else }}
-  clusterIP: None
+  type: ClusterIP
```

Comment:
`clusterIP: None` is the current setup in starship and I believe we would need good reasoning for changing it.

Reply:
Ah I remember now, this is a `headless` service: https://kubernetes.io/docs/concepts/services-networking/service/#headless-services

We could see how they feel about it. My guess is that it is `None` because they didn't know how to expose the explorer service (due to the embedded chain rest/rpc apis in the page which need to be routable too).

Maybe we'd need to explain our approach to this problem and see if they agree with how we are doing it.